docs/build/filecoin-pinning-services.md

```diff
@@ -40,6 +24,14 @@ While each FPS provider will have slightly different instructions for utilizing
 - Maintain and monitor your data storage.
 - Retrieve your data through the API of the service you are using.
 
+## Key benefits of FPSs
+
+1. Data backed up by decentralized storage network (the Filecoin network) for data persistence.
+
+2. Perfect for developers that do not want to worry about maintaining their own decentralized infrastructure while avoiding vendor lock-in. All providers contribute to the same storage network.
```

Comment:
```suggestion
2. Developers do not need to worry about maintaining their own decentralized infrastructure. Using an FPS also avoids _vendor lock-in_, since all providers contribute to the same storage network.
```